### PR TITLE
delete realization complex of module 35_transport

### PR DESCRIPTION
**Pull request by LaviniaBaumstark**
## Purpose of this PR
cleaning: delete realization complex of module 35_transport and all related code parts

## Type of change

- [x] Refactoring

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [x] All automated model tests pass (`FAIL 0` in the output of `make test`)
- [x] The changelog `CHANGELOG.md` [has been updated correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog)

## Further information (optional):

* Test runs are here: 
* Comparison of results (what changes by this PR?):

### CODE REVIEW

File: core/sets.gms [hunk 1]
```diff
@@ -343,6 +331,14 @@ $ENDIF.WindOff
         seceaf       "Route: Scrap-loaded EAF"
         pcc          "outdated technology, only here to avoid compilation errors if input data containing information for this technology are used"
         pco          "outdated technology, only here to avoid compilation errors if input data containing information for this technology are used"
+*** transport technologies for deleted realization complex of module 35_transport - only here for input data consistency
```
Comment (robertpietzcker) [inline core/sets.gms:334]: maybe not "input data consistency", but rather "to make it possible to process input data that still includes data for these obsolete transport technologies"?

**Review by robertpietzcker — APPROVED** [review]

File: main.gms [hunk 2]
```diff
@@ -1448,18 +1437,18 @@ $setglobal cm_incolearn  off !! def = off
 $setglobal cm_storageFactor  off !! def = off
 *** cm_learnRate "change learn rate value by technology."
 ***   def <- "off" = no change for learn rate value;
-***   or list of techs to change learn rate value. (ex. "apcarelt 0.2")
+***   or list of techs to change learn rate value.
```
Comment (Renato-Rodrigues) [inline main.gms:1440]: I would keep an example here, but use another tech, e.g. spv

File: main.gms [hunk 3]
```diff
@@ -1448,18 +1437,18 @@ $setglobal cm_incolearn  off !! def = off
...
 *** cm_learnRate "change learn rate value by technology."
 ***   def <- "off" = no change for learn rate value;
-***   or list of techs to change learn rate value. (ex. "apcarelt 0.2")
+***   or list of techs to change learn rate value. 
 $setglobal cm_learnRate  off !! def = off
 *** cm_adj_seed and cm_adj_seed_cont "overwrite the technology-dependent adjustment cost seed value. Smaller means slower scale-up."
 ***   both swicthes have the same functionality, but allow more changes once the character limit of cm_adj_seed is reached.
 ***   def <- "off" = use default adj seed values.
-***   or list of techs to change adj_seed value. (ex. "apCarH2T=0.5,apCarElT=0.5,apCarDiEffT=0.25,apCarDiEffH2T=0.25")
+***   or list of techs to change adj_seed value.
```
Comment (Renato-Rodrigues) [inline main.gms:1445]: again, I would keep an example using other technologies.

File: main.gms [hunk 4]
```diff
@@ -1448,18 +1437,18 @@ $setglobal cm_incolearn  off !! def = off
...
 $setglobal cm_adj_seed  off
 $setglobal cm_adj_seed_cont  off
 *** cm_adj_coeff and cm_adj_coeff_cont "overwrite the technology-dependent adjustment cost coefficient. Higher means higher adjustment cost."
 ***   both swicthes have the same functionality, but allow more changes once the character limit of cm_adj_coeff is reached.
 ***   def <- "off" = use default adj coefficient values.
-***   or list of techs to change adj_coeff value. (ex. "apCarH2T=100,apCarElT=100,apCarDiEffT=200,apCarDiEffH2T=200")
+***   or list of techs to change adj_coeff value.
```
Comment (Renato-Rodrigues) [inline main.gms:1451]: again, I would keep an example using other technologies.

File: modules/35_transport/edge_esm/sets.gms [hunk 5]
```diff
@@ -153,10 +153,6 @@ CAMP_lscStrong
 
 EDGE_scenario(EDGE_scenario_all) "Selected EDGE-T scenario"
 
-*** sets for the reporting, to be consistent with *complex* realisation
-FE_Transp_fety35(all_enty) "FEs used in the transport module"  / fepet, fedie, feh2t, feelt, fegat/
-FE_Elec_fety35(all_enty)   "FE electricity sets (should be moved to core/sets asap)"  / feels, feelt /
```
Comment (Renato-Rodrigues) [inline modules/35_transport/edge_esm/sets.gms:158]: I guess you are probably changing the remind2 reporting to remove whatever code relies on these for the transport edge run. Just to double checking.

**Review by Renato-Rodrigues — CHANGES_REQUESTED** [review]

File: main.gms (hunk 3 above)
Comment (LaviniaBaumstark) [reply main.gms:1445]: please provide a good example. I don't know one

File: main.gms (hunk 4 above)
Comment (LaviniaBaumstark) [reply main.gms:1451]: same here

File: modules/35_transport/edge_esm/sets.gms (hunk 5 above)
Comment (LaviniaBaumstark) [reply modules/35_transport/edge_esm/sets.gms:158]: I checked if deleted/renamed parameters/variables are used in our reporting

File: main.gms (hunk 2 above)
Comment (LaviniaBaumstark) [reply main.gms:1440]: please provide a good example. I don't know one

File: core/sets.gms [hunk 6]
```diff
@@ -932,7 +920,6 @@ $endif.altFeEmiFac
 ***######################## R SECTION START (MODULES) ###############################
 *** THIS CODE IS CREATED AUTOMATICALLY, DO NOT MODIFY THESE LINES DIRECTLY
 *** ANY DIRECT MODIFICATION WILL BE LOST AFTER NEXT MODEL START
-*** CHANGES CAN BE DONE USING THE RESPECTIVE LINES IN scripts/start/prepare.R
```
[the remark this answers is not in the document]
Comment (LaviniaBaumstark) [reply core/sets.gms:935]: no, this was deleted automatically :-)

File: core/sets.gms (hunk 1 above)
Comment (LaviniaBaumstark) [reply core/sets.gms]: changed

File: main.gms (hunk 2 above)
Comment (robertpietzcker) [reply main.gms:1440]: (ex. "spv 0.2")

File: main.gms (hunk 3 above)
Comment (robertpietzcker) [reply main.gms:1445]: (ex "spv=1, tnrs = 0.1")

File: main.gms (hunk 4 above)
Comment (robertpietzcker) [reply main.gms:1451]: (ex. "gash2=1, hydro=0.1")

**Review by Renato-Rodrigues — APPROVED** [review]

**Pull request merged**